utils/modular_model_converter.py

```diff
@@ -1087,12 +1087,18 @@ def convert_modular_file(modular_file, old_model_name=None, new_model_name=None,
         for file, node in cst_transformers.files.items():
             if node != {}:
                 # Get relative path starting from src/transformers/
-                relative_path = re.search(
+                regex_relative_path = re.search(
                     f"{os.sep}(src{os.sep}transformers{os.sep}.*)", os.path.abspath(modular_file)
```

Comment:
```suggestion
                    f"(.*){os.sep}modular_.*.py", os.path.abspath(modular_file)
```
could you try something like this, this way we also have the correct path for `examples` ( a bit more general!)

Reply:
I used `rf"(src{os.sep}transformers{os.sep}.*|examples{os.sep}.*)"` as a pattern to make it a bit more general, but keep the relative path instead of the absolute path